.github/workflows/ci.yml

```diff
@@ -96,15 +101,15 @@ jobs:
       run: |
         pytest --cov=BLUEPRINT \
                --cov-report html:htmlcov_BLUEPRINT \
-               tests/BLUEPRINT --reactor
+               tests/BLUEPRINT --reactor --private
```

Comment:
don't think we're going to need any BLUEPRINT private tests? Also seems to be failing atm

Reply:
no it was just to make the testing fail sooner, I'll remove when it works